cmd/kubeadm/app/cmd/reset_test.go

```diff
@@ -149,7 +149,7 @@ func TestConfigDirCleaner(t *testing.T) {
 		if err != nil {
 			t.Errorf("Unable to create temporary directory: %s", err)
 		}
-		defer os.RemoveAll(tmpDir)
+		os.RemoveAll(tmpDir)
```

Comment:
here this is wrong. it will remove the dir before it is going to use it. I can understand it if this directory will be removed in the end of the loop, but well, not before `os.Mkdir(... tmpDir)`

Reply:
Thanks for your remind! I forget to move to it to the end of the outer loop. @kad